packages/java-packages/snippet-replacer-maven-plugin/src/test/resources/project-to-test/basic_src_snippet_parse.txt

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * This class contains code samples for generating javadocs through doclets for {@link ConfigurationClient}
+ */
+public final class ConfigurationClientJavaDocCodeSnippets {
+
+    private String key1 = "key1";
+    private String key2 = "key2";
+    private String value1 = "val1";
+    private String value2 = "val2";
+
+    /**
+     * Generates code sample for creating a {@link ConfigurationClient}
+     *
+     * @return An instance of {@link ConfigurationClient}
+     * @throws IllegalStateException If configuration credentials cannot be created.
+     */
+    public ConfigurationClient createAsyncConfigurationClientWithPipeline() {
+
+    /**
+     * Generates code sample for creating a {@link ConfigurationClient}
+     *
+     * @return An instance of {@link ConfigurationClient}
+     * @throws IllegalStateException If configuration credentials cannot be created
+     */
+    public ConfigurationClient createSyncConfigurationClient() {
+        String connectionString = getConnectionString();
+        // BEGIN: com.azure.data.applicationconfig.configurationclient.instantiation
```

Comment:
It would be good to also include a test case for a snippet that has line and block comments. Also, add some snippets that have `-> ` that should be encoded.

Reply:
Great ideas. Will add test cases for this.

Reply:
Should cover the spectrum now.